io.github.gssh_linux.gssh.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.gssh_linux.gssh
+runtime: org.gnome.Platform
+runtime-version: '44'
+runtime-options:
+sdk: org.gnome.Sdk
+command: runner.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --env=GI_TYPELIB_PATH=/app/lib/girepository-1.0
+modules:
+  - name: Gssh
+    buildsystem: simple
+    build-commands:
+      - install -D Flatpak/runner.sh /app/bin/runner.sh
+      - install -D main.py /app/bin/main.py
+      - install -D Flatpak/.desktop /app/share/applications/io.github.gssh_linux.gssh.desktop
+      - install -D Flatpak/appicon.png /app/share/icons/hicolor/128x128/apps/io.github.gssh_linux.gssh.appicon.png
```

Comment:
Icon name doesn't match the app-id.

Reply:
It is valid but a bit unusual naming.

Reply:
Also general recommendation, these files have *nothing* to do with flatpak. They are files that will always be installed for every package. So it is strange to just shove them in a `Flatpak` directory.

Reply:
it'll get exported and the desktop file reference it like that. But I'm also of the opinion that this is a bad practice. And I'm not sure if it doesn't have any obscure side effect down the line.

Also I recommend naming the desktop file properly in the upstream repository. `.desktop` is a bad idea as it starts with a `.`.

Reply:
also that image is 512x512 so its install location is incorrect.